llvm/include/llvm/MC/MCObjectStreamer.h

```diff
@@ -90,6 +90,12 @@ class MCObjectStreamer : public MCStreamer {
 
   MCFragment *getCurrentFragment() const;
 
+  template <typename F, typename... Args> F *allocAndAdd(Args &&...args) {
+    F *Frag = new F(std::forward<Args>(args)...);
```

Comment:
MCObjectStreamer.h cannot include MCContext.h, so we cannot use allocFragment.
I believe in the end we will have different allocation functions. The primary one from MCObjectStreamer.h will be made to a bump allocator., but `operator new` might remain.

Reply:
Hmm. What was the reason to put allocation inside the streamer? My thinking was to keep allocation in MCContext (maybe without merging allocation and adding), because it already manages all other resources, and then all fragments can go to the same allocator. Mixing bump ptr allocator and operator new raises the question makes it more difficult to decide how to free a fragment.

Reply:
Agree that allocAndAdd is not useful. Removed.